### PR TITLE
adds an example that uses both Redux and React Context API

### PR DESCRIPTION
**Pull request by jchiatt**
I'm not sure what the contributing guidelines are, so please let me know if I should add anything else. 😃

### CODE REVIEW

**Comment by GantMan** [comment]
Looks like you need to update the readme

image

**Comment by jchiatt** [comment]
@GantMan I fixed the issue - let me know if there's anything else I need here. 😃

File: README.md [hunk 1]
```diff
@@ -35,8 +35,10 @@
...
+| [React 16.x Context](https://reactjs.org/docs/context.html) + [Redux](https://github.com/reactjs/react-redux)                       | [<img src="./_art/octo.png" width="36px" />](https://github.com/GantMan/ReactStateMuseum/tree/master/React/redux-and-context) |                Coming Soon - Contribute Here?                |
 | [ReComponent](https://github.com/philipp-spiess/react-recomponent)                       | [<img src="./_art/csb.png" width="40px" />](https://codesandbox.io/s/github/GantMan/ReactStateMuseum/tree/master/React/react-recomponent) [<img src="./_art/octo.png" width="36px" />](https://github.com/GantMan/ReactStateMuseum/tree/master/React/react-recomponent) |                Coming Soon - Contribute Here?                |
 | [undux](https://github.com/bcherny/undux)                       | [<img src="./_art/csb.png" width="40px" />](https://codesandbox.io/s/github/GantMan/ReactStateMuseum/tree/master/React/undux) [<img src="./_art/octo.png" width="36px" />](https://github.com/GantMan/ReactStateMuseum/tree/master/React/undux) |                Coming Soon - Contribute Here?                |
+>>>>>>> a6e8515d2f7bc8daf437fd43072128ad6467a57d
```
Comment (GantMan) [inline README.md]:
```suggestion

```

Looks like there's a merge artifact still in the Readme.  Best to remove.

Comment (jchiatt) [reply README.md]: Whoops! Didn't catch that. Should be gone now. 😄

**Comment by GantMan** [comment]
awesome!  Thanks for the contribution!  Welcome to the contributors :)

**Pull request merged**